internals/maintenance_scripts.py

```diff
@@ -889,6 +889,26 @@ def get_template_data(self, **kwargs):
 class GenerateReviewActivityFile(FlaskHandler):
   """Generate a CSV file with all review activity in ChromeStatus."""
   DATE_FORMAT = '%Y-%m-%dT%H:%M:%S'
+  VOTE_VALUE_MAPPING: dict[str, SkyhookDashStatus] = {
+      'na': SkyhookDashStatus.FYI,
+      'review_requested': SkyhookDashStatus.PENDING_REVIEW,
+      'review_started': SkyhookDashStatus.PENDING_REVIEW,
+      'needs_work': SkyhookDashStatus.NEEDS_WORK,
+      'approved': SkyhookDashStatus.APPROVED,
+      'denied': SkyhookDashStatus.DENIED,
+      'na (self-certified)': SkyhookDashStatus.FYI,
+      'na (verified)': SkyhookDashStatus.FYI,
+      'no_response': SkyhookDashStatus.NOT_ASSIGNED_TO_LAUNCH_OWNER,
```

Comment:
`no_response`, `na_requested`, and `internal_review` still need to be mapped properly here.

Reply:
Let's try:
no_response: SkyhookDashStatus.PENDING_REVIEW,
na_requested: SkyhookDashStatus.PENDING_REVIEW,
internal_review: SkyhookDashStatus.PENDING_REVIEW,